tensorflow_serving/model_servers/main.cc

```diff
@@ -101,6 +101,16 @@ using tensorflow::serving::PredictionService;
 
 namespace {
```

Comment:
Can you please add test coverage of multiple models to tensorflow_model_server_test.py.